Collection Rate Analysis

carter-engen opened the pull request:
I created a .txt file with my analysis of Collection Rates plots from Analysis.R for your review.

carter-engen committed:
Collection Rate Individual Addition

I added a section of individual collection rate analysis to the end of Explore.R using previous Correct Item Collection Rate Analysis code.

carter-engen committed:
Collection Rate Additional Changes

Additional changes from 3/28 to add collection rate individual analysis.

carter-engen committed:
Added Name

Added full name to comment section of line 473.

carter-engen committed:
Adding CollectionRate Analysis

I created a .txt file with my analysis of collection rate graphs (plots at end of Analysis.R) for Jamiahus to review.

jamiahuswalton commented on CollectionRate_Analysis_CKE.txt:
```diff
@@ -0,0 +1,24 @@
+The data show a slightly normal distribution throughout, which most distributions centered around a rate of 55-65.
+Ind_Team and Ind show more similar distributions, whereas Team shows a slightly more flattened distribution.
+
+Collection Rate v. Target by Strategy Plot:
```
Before to include the type of plot (Bar plot, Interaction plot, etc.). 

Also please include which level you are referring to (i.e., Individual or Team).

Do this for all of the sections.

jamiahuswalton commented on CollectionRate_Analysis_CKE.txt:
```diff
@@ -0,0 +1,24 @@
+to this is the similarity between Ind/Alone and Team/Together strategies, which seem to have the lowest collection rates of
+all strategies (and are nearly identical in rate).
+
+Collection Rate v. SessionOrder by Target:
```
Note sure which graph to look at. Please include the graph type

jamiahuswalton reviewed (changes requested):
Overall, the text looks good but it is a little hard to follow because I am not sure which graph to look at while I am reading your text. 

I believe I told you to create a separate file for your analysis but I was wrong. I think it would make it easier to follow if you can the .txt file with the explore file. Take the text for a particular plot and add it right under the code for the plot in the explore.r file. 

Let me know if this does not make sense.

carter-engen committed:
Analysis added within R Code

I have added my Collection Rate Analysis within the Explore.R file to show which plots are being referenced.

jamiahuswalton reviewed (approved):
Thanks, It is much easier to read.

jamiahuswalton commented:
@carterengen , if you are done with this branch, please delete it. 

I like deleting branches that we are not using because it keeps the repo clean and organized.
